zebra/zebra_vty.c

```diff
@@ -2444,6 +2493,7 @@ DEFPY(ipv6_route_address_interface_vrf,
             tag (1-4294967295)                             \
             |(1-255)$distance                              \
             |label WORD                                    \
+	    |table (1-4294967295)                          \
```

Comment:
Remove as above.

Reply:
this parameter will be handled only in the case with vrf backend = VRF netns; That parameter will be visible but not authorised if running zeba with default vrf backend=  VRF lite.
This parameter will be kept then.